sources/sections-a1-en/02-3rd-cgpm.adoc

```diff
@@ -1,19 +1,25 @@
+[[cgpm3rd1901]]
 == 3^rd^ CGPM, 1901
 
+[[cgpm3rd1901litre]]
 === Declaration concerning the definition of the litre (CR, 38-39)
 
+____
 &#x200c;. . .
 
 *The Conference declares*
 
 . The unit of volume, for high accuracy determinations, is the volume occupied by a mass of 1 kilogram of pure water, at its maximum density and at standard atmospheric pressure: this volume is called "litre".
 
 . . . .
+____
 
-NOTE: This definition was abrogated in 1964 by the 12^th^ CGPM (Resolution 6, see p. 167).
+NOTE: This definition was abrogated in 1964 by the <<cgpm12th1964r6,12^th^ CGPM (Resolution 6)>>.
```

Comment:
@manuel489 this has to be encoded as `12^th^ CGPM (<<cgpm12th1964r6,Resolution 6>>)` in accordance with BIPM's advice [here](https://github.com/metanorma/metanorma-bipm/issues/17#issuecomment-704645857).

This is so that the rendered form will be like this on PDF: `12^th^ CGPM (<link>Resolution 6, see p. 167</link>)`.

Reply:
Done.